orchestration/src/main/java/com/sap/ai/sdk/orchestration/AssistantMessage.java

```diff
@@ -0,0 +1,17 @@
+package com.sap.ai.sdk.orchestration;
+
+import javax.annotation.Nonnull;
+import lombok.Value;
+import lombok.experimental.Accessors;
+
+/** Represents a chat message as 'assistant' to the orchestration service. */
+@Value
```

Comment:
This implicitly creates public getters. To avoid breaking changes in the future, we could make them package-private for now. Alternatively, we could override the getter of content to return `Object` instead of string

Reply:
on the other hand I guess we need it to be public for response evaluation and for implementing the interface.. so yeah, not sure what we can do about that